manifests/x/XnSoft/XnViewMP/0.99.1/XnSoft.XnViewMP.locale.en-US.yaml

```
@@ -0,0 +1,25 @@
+PackageIdentifier: XnSoft.XnViewMP
+PackageVersion: 0.99.1
+PackageLocale: en-US
+Publisher: Gougelet Pierre-e
+PublisherUrl: https://www.xnview.com/en/
+PublisherSupportUrl: https://www.xnview.com/en/xnviewmp/#support
+PrivacyUrl: https://www.xnview.com/PrivacyPolicy.html
+Author: Gougelet Pierre-Emmanuel
+PackageName: XnViewMP
+PackageUrl: https://www.xnview.com/en/xnviewmp/
+License: GFL SDK/GflAx
+LicenseUrl: https://www.xnview.com/wiki/index.php/License
+Copyright: Copyright (c) 1991-2021 Pierre-e Gougelet
+CopyrightUrl: https://www.xnview.com/wiki/index.php?title=XnView_User_Guide
+ShortDescription: The enhanced Image Viewer for (Windows/MacOS/Linux)
+Moniker: xnviewmp
+Tags:
+- image
+- image viewer
```

Someone gonna suggest this sooner or later 👀
```suggestion
- image-viewer
```